sources/packages/web/src/helpers/utilfunctions.ts

```diff
@@ -24,4 +26,15 @@ export default class Helper {
     const formattedDate = new Date(date);
     return formattedDate.toLocaleDateString("en-CA", options);
   }
+
+  static addressLineItemList(address: Address) {
```

Comment:
Please move the method to the useFormatters composable.
This helper file should be killed IMO.